tests/unit/test_utils.py

```diff
@@ -218,7 +218,7 @@ def mock_version(path):
     @pytest.mark.windows
     @pytest.mark.skipif(os.name != "nt", reason="Windows test only")
     def test_windows_shellquote(self):
-        test_path = "C:\Program Files\Python36\python.exe"
+        test_path = r"C:\Program Files\Python36\python.exe"
```

Comment:
Based on the presence of 4 backslashes in the original, I'm guessing you'd need 2 backslashes in a raw string and then this should work